app/controllers/items_controller.rb

```diff
@@ -48,37 +51,61 @@ def create                                  #function meant to handle the creati
   end 
 
   def update                                   #function that handles updating library items. 
-   @item = Item.find(params[:id])
-   if @item.update(item_params)
+    @item = Item.find(params[:id])
+    if @item.update(item_params)
       redirect_to @item
-   else
+    else
       render 'edit'
-   end
+    end
   end
  
   def destroy                                 #function that handles destruction of library items. 
     @item = Item.find(params[:id])
     @item.destroy
-    redirect_to items_path
+    redirect_to 'items'
   end
  
+  # Modifying status
+  def deny
+     @item = Item.find(params[:id])
+     @item.update_attribute(:status, Item::DENIED)
+     
+     redirect_to :action => 'index'
+  end
+  
+  def approve
+    @item = Item.find(params[:id])
+    @item.update_attribute(:status, Item::APPROVED)
+    
+    redirect_to :action => 'index'
+  end
+
   private
+  # find all pending items in the database
+  def find_pending
```

Comment:
Is this being used anywhere? Also not seeing `find_approved` being used.

Reply:
I think I don't use them anymore, I will remove them to avoid confusion.